pkg/sdkserver/localsdk.go

```diff
@@ -409,8 +409,8 @@ func (l *LocalSDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerID)
 		return &alpha.Bool{}, errors.New("Players are already at capacity")
 	}
 
-	l.gs.Status.Players.IDs = append(l.gs.Status.Players.IDs, id.PlayerID)
-	l.gs.Status.Players.Count = int64(len(l.gs.Status.Players.IDs))
+	l.gs.Status.Players.Ids = append(l.gs.Status.Players.Ids, id.PlayerID)
+	l.gs.Status.Players.Count = int64(len(l.gs.Status.Players.Ids))
 
 	l.update <- struct{}{}
 	l.recordRequestWithValue("playerconnect", "1234", "PlayerIDs")
```

Comment:
Is it worth keeping everything consistent, e.g. this comment, the loop variable playerID -> playerId?